exporter/tanzuobservabilityexporter/README.md

```diff
@@ -61,6 +61,11 @@ reduce memory
 consumption. [Details and defaults here](https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/memorylimiterprocessor/README.md)
 .
```

Comment:
We need to delete the newline. If you look at how the markdown renders this sentence [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/tanzuobservabilityexporter#advanced-configuration), you can see it add a space between the link and the period.

Reply:
Fixed.